code/game/objects/items/stacks/stack.dm

```diff
@@ -201,31 +201,31 @@
 			to_chat(usr, "<span class='warning'>\The [R.title] won't fit here!</span>")
```

Comment:
You probably want to change the argument here for line 200 as well, since it has the same issue.

Reply:
Please make this change